ga4gh/datamodel/references.py

```diff
@@ -5,3 +5,56 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+
+import os
+import glob
+
+import pysam
+
+import ga4gh.protocol as protocol
+
+
+class ReferenceSet(object):
+    """
+    TODO document.
+    """
+    def __init__(self, id_, dataDir):
+        self._id = id_
```

Comment:
Not sure if it would be worth deriving all of these objects from a common superclass.  They all seem to have an id, for instance... something to consider.